src/js/sa11y.js

```diff
@@ -17,7 +17,7 @@ import Elements from './utils/elements';
 
 // Extras
 import detectPageChanges from './logic/detect-page-changes';
-import { dismissAnnotationsLogic, dismissAnnotationsButtons } from './logic/dismiss-annotations';
+import { dismissAnnotationsLogic, dismissAnnotationsButtons, getResultsForAJAX } from './logic/dismiss-annotations';
```

Comment:
where is this new function?

Reply:
Upon commit, I noticed that this function was not used and I reverted the changes, but forgot to remove this reference. Leaving this out is perfectly fine. In particular with your changes.